go/ngql/types.go

```diff
@@ -439,9 +439,9 @@ func getListElements(v types.Value, args map[string]interface{}) interface{} {
 	var iter listOrSetIterator
 	switch l := l.(type) {
 	case types.List:
-		iter = l.IteratorAt(uint64(idx))
+		iter = l.IteratorAt(uint64(idx), false)
 	case types.Set:
-		iter = l.IteratorAt(uint64(idx))
+		iter = l.IteratorAt(uint64(idx), false)
```

Comment:
I don't think there's a good way to pick a constant. GQL is the easy case here since we know the exact begin/end point. I think the thing to do for GQL is to add API which allows you fetch ranges and then we can load those optimally.